samples/constants.go

```diff
@@ -5,8 +5,18 @@ import "fmt"
 
 const Phi = 2.61803398875
 
+// The `iota` identifier is used in const declarations to
+// simplify definitions of incrementing numbers.
```

Comment:
would it be too off-topic / confusing to reference other languages' enums as a more tangible example of why you'd use such a thing?

Reply:
Yeah I thought about that too, I haven't referenced other languages elsewhere though...
Adding it to the speaker notes, if that's alright.